docs/src/examples/examples_barchartshistograms.md

```diff
@@ -185,7 +185,7 @@ using VegaLite, DataFrames
 
 data = DataFrame(
     question=["Question $(div(i,5)+1)" for i in 0:39],
-    type=repeat(["Strongly disagree", "Disagree", "Neither agree nor disagree",
+    typ=repeat(["Strongly disagree", "Disagree", "Neither agree nor disagree",
```

Comment:
We don't need to do this anymore, `type` is no more a reserved keyword, so we can just leave as is!